docs/section-7/feedback-control.md

```diff
@@ -55,10 +55,13 @@ double calculate(double currentPosition, double targetPosition) {
     return kP * error + accumulatedError + kD * derivative;
 }
 ```
+!!! warning
```

Comment:
should be moved to be right under the header, this message is not something to be read AFTER I read the entire section and fail to understand the math

Reply:
I don't think it should be moved exactly under the header "Theory", if that's what you are talking about. If you want to utilize PID you have to understand the theory but you don't need to know the exact details of the implementation. I moved it right above the code that shows the exact implementation.